LeaderboardBackend/Models/Entities/Modship.cs

```diff
@@ -34,6 +34,7 @@ public class Modship
 	/// <summary>
 	///     Relationship model for `LeaderboardId`.
 	/// </summary>
+	[JsonIgnore]
 	public Leaderboard? Leaderboard { get; set; } = null!;
```

Comment:
This is because I wanted to remove all references to the Leaderboard entity from the `openapi.json` spec when I created the Leaderboard view models; and I was too lazy to create a Modship view model because we won't have moderators in MVP.

Reply:
Fair. And there's definitely no point in creating the Modship view model anyway; no worries about "being lazy" about that :+1:

Reply:
Could you add a comment there explaining the reason for it then? So we remember